### DISCUSSION TITLE
Shipping update?

### DISCUSSION THREAD

**Discussion by TheCraiggers** [micro-nova/AmpliPi#183] [General] [answered]
I unfortunately missed the kickstarter but ordered off your site a couple months ago. My Amplipi was supposedly going to ship Sept. 30th; well, not so much ship since I'm picking it up in person. Anyway. just curious about an update since I haven't heard anything since placing the order.

Thanks!

**Comment by gorski123** [comment, marked as answer]
Hey Craig,
We had a few issues with our chassis manufacturer, they were unable to deliver as scheduled due to staffing issues. We also had a few issues sourcing some key components for the build. The good news is we have made it through both of those issues and are now building up and shipping units! We just shipped a batch of 10 Kickstarter orders today. Our goal is to get all the pre-orders that were due in Sept out by the end of this month. I am really sorry for the delay, I think we underestimated the amount of time it would take to ramp up manufacturing.

**Reply by TheCraiggers** [reply]
Thanks for the update and the email!
